src/lava/proc/dense/models.py

```diff
@@ -49,42 +50,40 @@ class PyDenseModelBitAcc(PyLoihiProcessModel):
 
     s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, bool, precision=1)
     a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.int32, precision=16)
-    a_buff: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=16)
+    a_buff: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=16)
     weights: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=8)
-    weight_exp: np.ndarray = LavaPyType(np.ndarray,np.int32, precision=4)
-    num_weight_bits: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=3)
-    sign_mode: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=2)
+    weight_exp: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=4)
+    num_weight_bits: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=3)
+    sign_mode: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=2)
 
     def __init__(self):
-        super(PyDenseModelBitAcc,self).__init__()
-        #Flag to determine whether weights have already been scaled.
+        super(PyDenseModelBitAcc, self).__init__()
+        # Flag to determine whether weights have already been scaled.
         self.weights_set = False
 
     def _set_wgts(self):
```

Comment:
Since there's no learning. This method can just be called during __init__. Even the check for weigths_set during run_spk is unnecessary.

Reply:
> 

We cannot do things in `__init__` of a ProcessModel. The variables are all `LavaPyType`s when we begin. No real computation can happen with these.